scripts/bridge/setup/from-config/cmd/setup-from-config.go

```diff
@@ -50,20 +50,35 @@ func main() {
 	accountID := flag.String("accountID", "0.0", "Hedera Account ID")
 	network := flag.String("network", "testnet", "Hedera Network Type")
 	members := flag.Int("members", 1, "The count of the members")
+	memberPrivateKeys := flag.String("memberPrivateKeys", "", "Member private keys array, seperated by ','")
 	adminKey := flag.String("adminKey", "", "The admin key")
 	topicThreshold := flag.Uint("topicThreshold", 1, "Topic member keys sign threshold")
 	wrappedFungibleThreshold := flag.Uint("wrappedTokenThreshold", 1, "The desired threshold of n/m keys required for supply key of wrapped tokens")
 	configPath := flag.String("configPath", "scripts/bridge/setup/extend-config/extended-bridge.yml", "Path to the 'bridge.yaml' config file")
 	flag.Parse()
 
+	prKeysSlice := strings.Split(*memberPrivateKeys, ",")
+	var hederaPrivateKeys []hedera.PrivateKey
+	for i := 0; i < len(prKeysSlice); i++ {
+		// element [0] can be empty string if the user does not provide any private keys
+		if prKeysSlice[0] == "" {
+			break
+		}
```

Comment:
I think this requires some mind gymnastics to understand, so we should either:
1. Surround the for with the if instead
2. Extract this logic in a function which returns the private keys array from the string and do:
```golang
if prKeysSlice[0] == "" {
	return nil
}
```

Reply:
Noted and refactored to:
```
	// element [0] can be empty string if the user does not provide any private keys
	if prKeysSlice[0] != "" {
		for i := 0; i < len(prKeysSlice); i++ {
			privateKeyFromStr, err := hedera.PrivateKeyFromString(prKeysSlice[i])
			if err != nil {
				panic(err)
			}
			hederaPrivateKeys = append(hederaPrivateKeys, privateKeyFromStr)
		}
	}
```